src/components/app/ListOfPublishedProfiles.js

```diff
@@ -358,13 +358,21 @@ export class ListOfPublishedProfiles extends PureComponent<Props, State> {
             />
           ))}
         </ul>
-        {profilesRestCount > 0 ? (
+        {withActionButtons ? null : (
           <p>
             <InnerNavigationLink dataSource="uploaded-recordings">
-              See and manage all your recordings ({profilesRestCount} more)
+              {profilesRestCount > 0 ? (
+                <>
```

Comment:
This is kind of surprising to use fragments, but the alternative is to use strings so I think it's still better.